sdk/spring/spring-cloud-azure-feature-management/src/test/resources/validations-tests/TimeWindowFilter.tests.json

```diff
@@ -0,0 +1,57 @@
+[
+    {
+        "FeatureFlagName": "PastTimeWindow",
+        "Inputs": {},
+        "IsEnabled": {
+            "Result": "false"
+        },
+        "Variant": {
+            "Result": null
+        },
+        "Description": "Time Window filter where both Start and End have already passed."
+    },
+    {
+        "FeatureFlagName": "FutureTimeWindow",
+        "Inputs": {},
+        "IsEnabled": {
+            "Result": "false"
+        },
+        "Variant": {
+            "Result": null
+        },
+        "Description": "Time Window filter where neither Start nor End have happened."
+    },
+    {
+        "FeatureFlagName": "PresentTimeWindow",
+        "Inputs": {},
+        "IsEnabled": {
+            "Result": "true"
+        },
+        "Variant": {
+            "Result": null
+        },
+        "Description": "Time Window filter where Start has happend but End hasn't happend."
```

Comment:
happened spelled wrong twice, I've fixed it in the python repo. Not in the AppConfiguraiton one yet.

Reply:
I see, missing "e" for "happened". Added it.

Reply:
Send out a PR to "AppConfiguraiton" repo to fix this: https://github.com/Azure/AppConfiguration/pull/951

Reply:
Thanks for fixing that.